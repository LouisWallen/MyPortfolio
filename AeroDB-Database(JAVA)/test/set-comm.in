set a 5 5 8 3
set b 8 8 9
diff a b
union a b
inter a b
cartprod a b
bye
