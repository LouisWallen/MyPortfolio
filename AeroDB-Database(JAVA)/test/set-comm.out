> ok

> ok

> [3 5 9]

> [3 5 8 9]

> [8]

> [ [5 8] [5 8] [5 9] [5 8] [5 8] [5 9] [8 8] [8 8] [8 9] [3 8] [3 8] [3 9] ]

> bye
